pillar/heroku/ocw-studio.sls

```diff
@@ -56,7 +53,6 @@ heroku:
     {% set rds_endpoint = salt.boto_rds.get_endpoint('ocw-studio-db-applications-{}'.format(env_data.env)) %}
     DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/ocw_studio
     {% endif %}
-    GTM_ACCOUNT_ID: {{ env_data.GTM_ACCOUNT_ID }}
```

Comment:
I'm not clear on why this was removed from ocw-studio? I understand that the ocw-www is the main site, but don't want to analytics on the studio site as well?

Reply:
I think these accounts are for ocwnext, not ocw-studio - is that right @pdpinch?

Reply:
Sorry, the ones you have in this PR are specific to OCW Studio. You should have access to confirm this in https://tagmanager.google.com/#/home (ask me to add you if you don't). Anyway here's a screenshot: 

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/430126/107275841-412e8e80-6a20-11eb-9059-0a9825091f41.png">

I'll make up some tags for OCW Next, which will include the historical UTM tags for continuity.

Reply:
I put back the existing ocw-studio values, replaced ocw-www values with `?` for now